plugin-server/src/main/ingestion-queues/session-recording/session-recordings-blob-consumer.ts

```diff
@@ -201,31 +201,46 @@ export class SessionRecordingBlobIngester {
 
         this.offsetManager = new OffsetManager(this.batchConsumer.consumer)
 
-        this.batchConsumer.consumer.on('rebalance', (event) => {
-            /**
-             * This event is received when the consumer group _starts_ rebalancing.
-             * During the rebalance process, consumers within the group stop processing messages temporarily.
-             * They cannot receive or process events until the rebalance is completed
-             * and the new partition assignments have been made.
-             *
-             * Since that means session managers don't know they will still be assigned to a partition
-             * They must stop flushing sessions to S3 until the rebalance is complete.
-             */
-            status.debug('⚖️', 'Blob ingestion consumer rebalancing', { event })
-            this.sessions.forEach((session) => session.pauseFlushing())
-        })
-
-        // TODO: This used to be group_join - subscribed is probably not right.
-        this.batchConsumer.consumer.on('subscribed', (event) => {
+        // NOTE: Do we need to pause when the rebalancing starts??
```

Comment:
What I wanted to avoid was

1. rebalancing starts
2. enough time passes that the session manager flushes to S3
3. it can't commit its offsets
4. rebalancing finishes and it is no longer assigned to that partition
5. another consumer now writes to the same S3 file

Maybe that's safe 🤷 🙈